operator/buffer/memory.go

```diff
@@ -28,31 +30,113 @@ func NewMemoryBufferConfig() *MemoryBufferConfig {
 	}
 }
 
-// Build builds a MemoryBufferConfig into a Buffer, loading any entries that were previously unflushed
-// back into memory
+// Build builds a MemoryBufferConfig into a Buffer
 func (c MemoryBufferConfig) Build(operatorID string) (Buffer, error) {
-	return &MemoryBuffer{}, nil
+	return &MemoryBuffer{
+		operatorID:    operatorID,
+		buf:           make(chan *entry.Entry, c.MaxEntries),
+		maxChunkDelay: c.MaxChunkDelay.Raw(),
+		maxChunkSize:  c.MaxChunkSize,
+		closed:        false,
+	}, nil
 }
 
-// MemoryBuffer is a buffer that holds all entries in memory until Close() is called,
-// at which point it saves the entries into a database. It provides no guarantees about
-// lost entries if shut down uncleanly.
+// MemoryBuffer is a buffer that holds all entries in memory until Close() is called.
+// Once close is called all entries will be lost it is reccommended to call Drain before Close.
 type MemoryBuffer struct {
+	operatorID    string
```

Comment:
Is this actually used anywhere in this buffer? I can't seem to find a use; Seems like it was used as a unique key when storing to the database in the previous implementation

Reply:
That is a great question. I think you're right. Looks like the disk buffer doesn't use it either. I'll do a follow up PR to correct the interface again and fix the tests. That'll be easier than doing the interface update and rebasing.

Reply:
It might be necessary in the hybrid buffer, if we want to keep it for that.

Reply:
Actually, probably not. Makes sense to remove it.

Reply:
Yeah I agree. If we don't need it for disk or memory probably don't need it for a hybrid